### PR TITLE
CASM-2423 Add documentation for Grafterm tool

### PR DESCRIPTION
**Pull request by itsvijaybs**
This change is needed for: https://connect.us.cray.com/jira/browse/CASMMON-98 

image

### CODE REVIEW

File: operations/system_management_health/Grafterm.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Grafterm
+
+Visualize metrics dashboards on the terminal, like a simplified and minimalist version of **Grafana** for terminal.
+
+The utility(script) can be found in the /opt/cray/platform-utils directoty in all NCNs.
```
Comment (bklei) [inline operations/system_management_health/Grafterm.md:5]: 'directory'

File: operations/system_management_health/Grafterm.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ncn-m001# ./grafterm.sh --help
+```
+
+### List available terminal dashboards
+
+```bash
+ncn-m001# ./grafterm.sh --list
+```
+
+### simple
```
Comment (bklei) [inline operations/system_management_health/Grafterm.md:23]: Maybe 'Default Usage'?  Could also say what the default days and refresh settings are...

File: operations/system_management_health/Grafterm.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Grafterm
+
+Visualize metrics dashboards on the terminal, like a simplified and minimalist version of **Grafana** for terminal.
+
+The utility(script) can be found in the /opt/cray/platform-utils directoty in all NCNs.
+
+## Running options
+
+Exit with `q` or `Esc`
```
Comment (nrockershousen) [inline operations/system_management_health/Grafterm.md:9]: Add a period at the end.

File: operations/system_management_health/Grafterm.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+### List available terminal dashboards
+
+```bash
+ncn-m001# ./grafterm.sh --list
+```
+
+### simple
+To view the dashboard, pass the value(dashboard json file) to the -c parameter to the script.
```
Comment (nrockershousen) [inline operations/system_management_health/Grafterm.md:24]: Add backticks to `-c`

File: operations/system_management_health/Grafterm.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+ncn-m001# ./grafterm.sh -c critical_services_dashboard.json -r 10s
+```
+
+```bash
+ncn-m001# ./grafterm.sh -c critical_services_dashboard.json  -d 3h -r 10s
+```
+
+### Fixed time
+
+Setting a fixed time range to visualize the metrics using duration notation. In this example is start at `now-23h` and end at `now-18h`
```
Comment (nrockershousen) [inline operations/system_management_health/Grafterm.md:47]: - Change "setting" to "set"
- Change "In this example is start at `now-23h` and end at `now-18h`" to "In the following example, the start time is `now-23h` and the end time is `now-18h`."

File: operations/system_management_health/Grafterm.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+```bash
+ncn-m001# ./grafterm.sh -c critical_services_dashboard.json -s 23h -e 18h
+```
+
+Setting a fixed time range to visualize the metrics using timestamp [ISO 8601] notation.
```
Comment (nrockershousen) [inline operations/system_management_health/Grafterm.md:53]: Change "Setting" to "Set"

File: operations/system_management_health/Grafterm.md (hunk 1 above)
Comment (itsvijaybs) [reply operations/system_management_health/Grafterm.md]: > 'directory'

Addressed

File: operations/system_management_health/Grafterm.md (hunk 2 above)
Comment (itsvijaybs) [reply operations/system_management_health/Grafterm.md]: > Maybe 'Default Usage'? Could also say what the default days and refresh settings are...

Addressed. Thanks!

File: operations/system_management_health/Grafterm.md (hunk 3 above)
Comment (itsvijaybs) [reply operations/system_management_health/Grafterm.md]: > Add a period at the end.

Addressed

File: operations/system_management_health/Grafterm.md (hunk 4 above)
Comment (itsvijaybs) [reply operations/system_management_health/Grafterm.md]: > Add backticks to `-c`

addressed.

File: operations/system_management_health/Grafterm.md (hunk 5 above)
Comment (itsvijaybs) [reply operations/system_management_health/Grafterm.md]: Done.

File: operations/system_management_health/Grafterm.md (hunk 6 above)
Comment (itsvijaybs) [reply operations/system_management_health/Grafterm.md]: > Change "Setting" to "Set"

Addressed. Thanks!

**Review by bklei — APPROVED** [review]

**Review by denniswalker — APPROVED** [review]

**Pull request merged**